CMakeLists.txt

```diff
@@ -80,9 +81,13 @@ else()
     # to add search paths.
     find_package(SDL3 CONFIG REQUIRED)
 
-    find_package(iniparser CONFIG COMPONENTS static)
-    if (NOT TARGET iniparser-static)
-        find_package(iniparser REQUIRED MODULE COMPONENTS static)
+    if (ISLE_SHARED)
```

Comment:
We don't really care what iniparser is chosen. Let's create an intermediate interface target (e.g. `Isle::iniparser`) so you can avoid the many if/else's.

The option `ISLE_SHARED` is not really needed.